### PR TITLE
Update CHANGELOG for v0.6.4

### PR DESCRIPTION
**Pull request by crypto-matto**
### Additions
- Tectonic Support
- CRC20 token whitelisting
- Message Type support in transaction history for `Crypto.org chain` asset
- Revamp Transaction History Records data types
- Adjust Validator list ordering to better promote validators with Up Time >99.9% yet low Voting Power
- Updated App Icon

### Bug fixes
- Fix showing of finished unbonding delegation
- Fix missing of vesting account support

### CODE REVIEW

**Review by XinyuCRO — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
...
 
 *Released*
+## [v0.6.4] - 2022-01-13
+### Additions
+- Add Tectonic Support
+- Add CRC20 token whitelisting
+- Add Message Type in transaction list of Crypto.org chain asset
+- Revamp on Transaction History Records data types
+- Adjust Validator list ordering to better promote validators with Up Time >99.9% yet low Voting Power
+- Update App Icon
```
Comment (cdc-Hitesh) [inline CHANGELOG.md:15]:
```suggestion
- Updated App Icon
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 *Unreleased*
 
 *Released*
+## [v0.6.4] - 2022-01-13
+### Additions
+- Add Tectonic Support
+- Add CRC20 token whitelisting
+- Add Message Type in transaction list of Crypto.org chain asset
+- Revamp on Transaction History Records data types
```
Comment (cdc-Hitesh) [inline CHANGELOG.md:13]:
```suggestion
- Revamp Transaction History Records data types
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 *Unreleased*
 
 *Released*
+## [v0.6.4] - 2022-01-13
+### Additions
+- Add Tectonic Support
+- Add CRC20 token whitelisting
+- Add Message Type in transaction list of Crypto.org chain asset
```
Comment (cdc-Hitesh) [inline CHANGELOG.md:12]:
```suggestion
- Added `MessageType` in transaction list of Crypto.org chain asset
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 *Unreleased*
 
 *Released*
+## [v0.6.4] - 2022-01-13
+### Additions
+- Add Tectonic Support
```
Comment (cdc-Hitesh) [inline CHANGELOG.md:10]: I think we should remove `Add` from the list. as the category title implies that.

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 *Unreleased*
 
 *Released*
+## [v0.6.4] - 2022-01-13
+### Additions
+- Tectonic Support
+- CRC20 token whitelisting
+- Add Message Type in transaction list of Crypto.org chain asset
```
Comment (cdc-Hitesh) [inline CHANGELOG.md:12]:
```suggestion
- Message Type support in transaction history for `Crypto.org chain` asset
```

**Review by cdc-Hitesh — DISMISSED** [review]

**Pull request merged**